contrib/grinder/scripts/blueflood.py

```diff
@@ -0,0 +1,87 @@
+import random
+try: 
+  from com.xhaus.jyson import JysonCodec as json
+except ImportError:
+  import json
+from utils import *
+from net.grinder.script import Test
+from net.grinder.plugin.http import HTTPRequest
+
+class IngestThread(AbstractThread):
```

Comment:
This class can have a better name may be `ingest.py` because it makes only ingestion requests?